### DISCUSSION TITLE
How do I use this plugin?

### DISCUSSION THREAD

**Discussion by kevinlinxc** [trinodb/grafana-trino#207] [Q&A] [answered]
This is unfortunately the first plugin I'm using, and I am quite confused as to how to use it.

After running the docker command:
```
docker run -d -p 3000:3000 \
  -v "$(pwd):/var/lib/grafana/plugins/trino" \
  -e "GF_PLUGINS_ALLOW_LOADING_UNSIGNED_PLUGINS=trino-datasource" \
  --name=grafana \
  grafana/grafana-oss
```

I do not see Trino in the plugins or as an available data source when I visit localhost:3000. What am I doing wrong?

**Comment by kevinlinxc** [comment]
I couldn't figure out how to use the plugin with a local Grafana either - I think the README was rather unclear, I assumed it meant to clone/download a zip of this repository and put it in the plugins folder, but it actually meant to download a release and put it into the plugins folder. Probably obvious to you, but not to new users. 

I still don't know how to use the plugin in Docker - do I need to download and copy the release into the Docker image, or is the docker command supposed to do it all?

**Reply by nineinchnick** [reply]
You have to download the release zip, extract it and put that in the plugins dir or build the plugin locally and put the dist dir in the plugins dir. I'll update the instructions soon. We're also close to getting the plugin certified so it'll make it much easier to run it and I'll also make sure it's documented.

**Reply by kevinlinxc** [reply]
I see, thank you. When do you think the plugin will be certified (50% sure date and 90% sure date?)

**Reply by nineinchnick** [reply, marked as answer]
It has been certified a few days ago.
